console/console.go

```diff
@@ -95,7 +98,16 @@ func GetConsolePluginCR(pluginName string, displayName string, consolePort int,
 //+kubebuilder:rbac:groups="",resources=services,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=console.openshift.io,resources=consoleplugins,verbs=*
 
-func InitConsole(client client.Client, odfPort int) error {
+func InitConsole(client client.Client, odfPort int ) error {
+	// The base path to where the request are sent
+	basePath := ""
+	clusterVersion, err  := util.DetermineOpenShiftVersion(client)
```

Comment:
I'll point out that this will only happen at operator startup. Thus, it seems entirely possible that the Console Pod itself would be on a version that is incompatible with our plugin until the odf-operator Pod is restarted. Is this the case?